File: src/main/kotlin/ympa_kotlin_wiremock_server/models/SetOrderBoxLayoutRequest.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class SetOrderBoxLayoutRequest(
    @field:JsonProperty("boxes")
    val boxes: kotlin.collections.List<OrderBoxLayoutDTO>,

    @field:JsonProperty("allowRemove")
    val allowRemove: kotlin.Boolean? = false,

)

File: lib/app/Models/SetOrderBoxLayoutRequest.php
<?php
/**
 * SetOrderBoxLayoutRequest
 */
namespace app\Models;

/**
 * SetOrderBoxLayoutRequest
 */
class SetOrderBoxLayoutRequest {

    /** @var \app\Models\OrderBoxLayoutDTO[] $boxes Список коробок.*/
    public $boxes = [];

    /** @var bool $allowRemove Передайте &#x60;true&#x60;, если вы собираетесь удалить часть товаров из заказа.*/
    public $allowRemove = false;

}
